integrationtest/backend/tck/src/main/java/org/hibernate/search/integrationtest/backend/tck/testsupport/types/ByteVectorFieldTypeDescriptor.java

```diff
@@ -73,6 +74,13 @@ protected List<byte[]> createNonMatchingValues() {
 		);
 	}
 
+	@Override
+	public List<byte[]> unitLengthVectors() {
```

Comment:
had to update how we were generating the sample data in both vector descriptors. Elastricsearch/OpenSearch both have more strict rules it seems and some of the values were not accepted with some of the similarity functions...

Reply:
created https://hibernate.atlassian.net/browse/HSEARCH-5048